src/common/datetime/format_date_time_with_seconds.ts

```diff
@@ -0,0 +1,23 @@
+import fecha from "fecha";
+
+// Check for support of native locale string options
+function toLocaleStringSupportsOptions() {
```

Comment:
We should generalize this and just export this from a new file. `export const toLocaleStringSupportsOptions = X`